docker/Dockerfile-build-aggregator.jvm

```diff
@@ -7,7 +7,7 @@ FROM registry.access.redhat.com/ubi8/openjdk-11:latest AS build
 USER root
 COPY . /home/jboss
 WORKDIR /home/jboss
-RUN mvn clean package -DskipTests -Dcheckstyle.skip=true --projects :checkstyle,:notifications-parent,:notifications-aggregator
+RUN mvn clean package --projects :checkstyle,:notifications-aggregator
```

Comment:
It would be even better if we could use `./mvnw` but I'm not sure it would work from a Docker build.

Reply:
It works \o/